modules/passkey/test/verifiers/WebAuthnVerifier.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import hre from 'hardhat'
+
+describe('WebAuthnVerifier', () => {
+  const setupTests = deployments.createFixture(async ({ deployments }) => {
+    const { WebAuthnVerifier } = await deployments.fixture()
+    const webAuthnVerifier = await ethers.getContractAt('WebAuthnVerifier', WebAuthnVerifier.address)
+    return { webAuthnVerifier }
+  })
+
+  it('Should return false when invalid signature', async () => {
```

Comment:
In this tests there are mutliple fields that are invalid:
- authenticator data
- clientDataFields
- (not sure here) x, y and r, s values are all zero, not sure if that's supported by the standard

Therefore it's unclear why the test fails. It may be because of the signature or because of any other field

Reply:
here, `WebAuthnVerifier` returns `false` in either of bloew:
a. Authenticator flag did not match
b. P256VerifierLib returned `false`

The test case sets `a` correctly so only case to have `false` return is `b`.

Reply:
This is not clear from the test description, I'd adjust it to `Should return false when the verifier returns false` in this case